README.md

```diff
@@ -1772,6 +1772,7 @@ Software for easy installation, management and configuration of self-hosted serv
 [Serverless computing - Wikipedia](https://en.wikipedia.org/wiki/Serverless_computing).
 
 - [Appwrite](https://appwrite.io) - End to end backend server for web, native, and mobile developers 🚀. ([Source Code](https://github.com/appwrite/appwrite)) `BSD-3-Clause` `PHP`
+- [Coolify](https://coolify.io/) - An open-source & self-hostable Heroku / Netlify alternative (and even more). ([Demo](https://demo.coolify.io/) [Source Code](https://github.com/coollabsio/coolify)) `Apache License 2.0` `TypeScript`
```

Comment:
Please add a missing comma to: 
`([Demo](https://demo.coolify.io/), [Source Code](https://github.com/coollabsio/coolify))`

Reply:
Done